tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+"""Unit tests for bring-api."""
+import os
+
+import aiohttp
+from dotenv import load_dotenv
+import pytest
+
+from bring_api.bring import Bring
+
+load_dotenv()
+
+
+@pytest.fixture
+async def session():
+    """Create  a client session."""
+    __session = aiohttp.ClientSession()
+    return __session
+
+
+@pytest.fixture
+async def bring(session):
+    """Create Bring instance."""
+    __bring = Bring(session, os.environ["EMAIL"], os.environ["PASSWORD"])
```

Comment:
Unit test **must** be isolated, there all responses from `aiohttp` have to be mocked and no real email/password is needed.

Reply:
Look for `patch` in relation with pytest, it lets you mock individual methods such as aiohttp.get, properties or entire objects such as aiohttp itself.

Reply:
yeah, babysteps xD
but emulating the bring api to have consistent test data is a good thing for the next iteration. There is also pytest-aiohttp 
 module that does that i think

Reply:
Cool, that sounds useful!